src/view_models/AccessGroupViewModel.js

```diff
@@ -62,6 +62,15 @@ export class TypeValueViewModel {
 }
 
 const defaultUserName = 'Anonymous User';
+const getDisplayName = (data) => {
```

Comment:
we should finally more https://github.com/mozilla-iam/dino-park-front-end/blob/master/src/components/search/SearchResult.vue#L48 into a shared function